core/tee/tee_obj.c

```diff
@@ -53,6 +53,25 @@ TEE_Result tee_obj_get(struct tee_ta_ctx *ctx, uint32_t obj_id,
 	return TEE_ERROR_BAD_PARAMETERS;
 }
 
+TEE_Result tee_obj_get1(struct tee_ta_ctx *ctx, uint32_t obj_id,
```

Comment:
"The TEE_TYPE_CORRUPTED_OBJECT is used solely in the deprecated TEE_GetObjectInfo function to indicate that the object on which it is being invoked has been corrupted in some way."
"Before this status is returned, the Implementation SHALL delete the corrupt object and SHALL close the associated handle; subsequent use of the handle SHALL cause a panic."
"TEE_ERROR_CORRUPT_OBJECT: If the persistent object is corrupt. The object handle is closed."
I should also add the deletion of the object  w/o access right restriction (TEE_CloseAndDeletePersistentObject/1)